src/main/java/com/eppo/sdk/helpers/ExperimentHelper.java

```diff
@@ -0,0 +1,12 @@
+package com.eppo.sdk.helpers;
+
+import java.util.Base64;
+
+public class ExperimentHelper {
+    static public String generateKey(
+            String flagKey,
+            String allocationKey
+    ) {
+        return flagKey + '-' + Base64.getEncoder().encodeToString(allocationKey.getBytes());
```

Comment:
🔥 love that we are base64 encoding!

Reply:
@aarsilv as per the recent discussion in the notion [doc](https://www.notion.so/Concatenated-experiment-key-design-d212aaaf42e7440bbdafd13cd0069dc8?d=900bdf54a4ac45cab39ac7b369772553&pvs=4#24eb8b56a36f4defbdf07eef352b3e28). I have removed the base 64 encoding and now returning `<flagkey>-<allocationkey>`

Reply:
Yeah actually looking over this again, I was wrong. `<allocationKey>` doesn't not base64 encode the allocation id

![Screenshot 2023-09-05 at 1 23 48 PM](https://github.com/Eppo-exp/java-server-sdk/assets/2822951/34d89eea-88d0-4b21-b6be-f58083a05ad9)

Ideally, all our ids would be hashids do users don't know how many experiments, feature flags, allocations, etc we are storing among all our customers at Eppo, but that ship sailed a long time ago.

What do you think @aarsilv ? Is the experiment key collision you mentioned in the Notion conversation something to worry about? I'm thinking it's okay if we just use the `allocationKey`, as is, e.g. `"allocation-819"` (that integer is still meaningful to the user, they'd see the same number in their URL when they edit the allocation) instead of `"allocation-QWxsb2NhdGlvbjoxNw=="`.

Reply:
Oh yeah, using numbers is fine, too, and I'm not worried about a collision in a practical sense.

True people could deduce our stats and usage from IDs, but I don't think that's a big deal. If, for whatever reason it becomes a big deal, we can always alter the tables to change the auto increment seed 😛